src/Domain/Messenger/GroupGateway.php

```diff
@@ -54,7 +54,9 @@ public function queryGroups(QueryCriteria $criteria, $gibbonPersonIDOwner = null
             ->leftJoin('gibbonGroupPerson', 'gibbonGroupPerson.gibbonGroupID=gibbonGroup.gibbonGroupID')
             ->leftJoin('gibbonPerson', 'gibbonPerson.gibbonPersonID=gibbonGroup.gibbonPersonIDOwner')
             ->groupBy(['gibbonGroup.gibbonGroupID']);
-
+        
+        $query->where('gibbonGroup.gibbonSchoolYearID = :gibbonSchoolYearID')
```

Comment:
The `->where()` can be chained into the query creation in this instance, because there's no conditional on it. Just code-style though 🤓, the changes look good and test out perfectly.